gh-pages/index.html

```diff
@@ -517,7 +517,7 @@
       isDisabled: false,
       isTouched: false,
       value: [],
-      options: ['Select option', 'Disable me!', 'Reset me!', 'mulitple', 'label', 'searchable']
+      options: ['Select option', 'Disable me!', 'Reset me!', 'multiple', 'label', 'searchable']
```

Comment:
The exclamation marks here may be emotional triggers to people who hate shouting. I'd advise you remove the exclamation marks or insert trigger warnings.

Reply:
omg ash stop following me around github commenting on my pull requests 😭

Reply:
Pull request, please! 👻